src/ceph_fuse.cc

```diff
@@ -85,13 +84,16 @@ int main(int argc, const char **argv, const char *envp[]) {
   auto cct = global_init(&def_args, args, CEPH_ENTITY_TYPE_CLIENT,
 			 CODE_ENVIRONMENT_DAEMON,
 			 CINIT_FLAG_UNPRIVILEGED_DAEMON_DEFAULTS);
-  for (std::vector<const char*>::iterator i = args.begin(); i != args.end(); ) {
+
+  auto i = args.begin();
+  auto end = args.end();
+  for (; i != end; ) {
```

Comment:
Instead:

`for (const auto &arg : args) {`

Reply:
`ceph_argparse_*` expect to get `iterator&`. How do you convert `const char*` to `iterator`?